fhirResourcesToLoad/o2billy_observation_hemocrit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "resourceType": "Observation",
+  "id": "pat015-hemocrit",
+  "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://hl7.org/fhir/observation-category",
```

Comment:
code system should be 'http://terminology.hl7.org/CodeSystem/observation-category'